packages/web3-core-subscriptions/src/subscription.js

```diff
@@ -310,6 +309,7 @@ Subscription.prototype.subscribe = function() {
                     _this.emit('error', error);
                 }
             });
+            _this.emit('connected', result);
```

Comment:
I don't get it, why we have to move this emitter outside the block? The `connected` event just implies that client is connected and should not be related to `this.options.requestManager.send`. I think we should remove that emitter, connection state should be managed, some where else regardless of `.send` for the request manager.